src/main/java/ch/njol/skript/conditions/CondCanBreed.java

```diff
@@ -0,0 +1,40 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.conditions.base.PropertyCondition;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import org.bukkit.entity.Breedable;
+import org.bukkit.entity.LivingEntity;
+
+@Name("Can Breed")
+@Description("Returns whether or not a living entity can be bred.")
```

Comment:
ditto

Reply:
what

Reply:
Replace return with check, ditto means its the same thing as my other comment